frontend/www/js/omegaup/components/user/ViewProfile.vue

```diff
@@ -342,7 +348,19 @@ export default class ViewProfile extends Vue {
 }
 </script>
 
-<style lang="scss" scoped>
+<style lang="scss">
```

Comment:
Acá se le podrá regresar el scoped
```suggestion
<style lang="scss" scoped>
```
?